bench/util/runner.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+const Autocannon = require('autocannon')
+const Bloomrun = require('bloomrun')
+const Minimist = require('minimist')
+
+// Our test config
+const Policies = require('../policies.bench.js')
+const Ping = require('../ping.bench.js')
+const Orgs = require('../orgs.bench.js')
+const Teams = require('../teams.bench.js')
+const Users = require('../users.bench.js')
+const List = require('../list.bench.js')
+const Access = require('../access.bench.js')
+
+// Apply each test handler to bloomrun with a tag
+function configureTests (tests) {
+  const container = Bloomrun()
+  
+  tests.forEach((route) => {
+    route.forEach((test) => {
+      container.add({tag:test.tag}, test.handler)
+    })
+  })
+
+  return container
+}
+
+// Prime the test suite, get the tag from cli, eg, `npm run bench -- get/orgs`
+const tests = configureTests([Ping, Orgs, Policies, Teams, Users, List, Access])
+const tag = (Minimist(process.argv.slice(2))._ || '').toString()
+
+// If we don't have a matching tag we exit early 
+const getParams = tests.lookup({tag:tag})
```

Comment:
@p16 Yup but generally I like to namespace them to avoid edge-cases and more versatility later. Example 'tag: x/y/x type1' where type is a subtype test for the same route.